Atomic/atomic.py

```diff
@@ -915,42 +917,26 @@ def _parse_imagename(imagename):
     def _convert_to_skopeo(image):
         i = image.replace("oci:", "")
         if "http:" in i:
-            return ["--insecure", "docker://" + i.replace("http:", "")]
+            return ["--insecure"], "docker://" + i.replace("http:", "")
```

Comment:
yes, it is hidden for now.  I can drop it, but I wanted to make a difference between `https:` (which is the default) and `http:` if specified